contrib/completion/zsh/_docker

```diff
@@ -2296,7 +2296,8 @@ __docker_swarm_subcommand() {
                 "($help)--listen-addr=[Listen address]:ip\:port: " \
                 "($help)--max-snapshots[Number of additional Raft snapshots to retain]" \
                 "($help)--snapshot-interval[Number of log entries between Raft snapshots]" \
-                "($help)--task-history-limit=[Task history retention limit]:limit: " && ret=0
+                "($help)--task-history-limit=[Task history retention limit]:limit: " && ret=0 \
+                "($help)--data-path-port=[Data Path Port] "
```

Comment:
I think this may not work, because the `&& ret=0` should be last.

Could you also move this up, to below `--data-path-addr` ? (if that works)?